ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/ipsec/file/provision/cucushift-installer-rehearse-azure-ipi-ovn-ipsec-file-provision-chain.metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+	"path": "cucushift/installer/rehearse/azure/ipi/ovn/ipsec/file/provision/cucushift-installer-rehearse-azure-ipi-ovn-ipsec-file-provision-chain.yaml",
```

Comment:
The keyword `file` is too common, not easy to understand, how about `filestore-csi`, or something else?

Reply:
The full name of the csi driver is `azurefile-csi-driver`, but seems azure is a bit duplicated, I'll rename as `file-csi` instead. Thanks!

Reply:
I think we can call it `azurefile` or `azurefile-csi`, `file-csi` sounds a bit strange.

Reply:
Sure, but the chain or workflow name already contains the `azure`, `cucushift-installer-rehearse-azure-ipi-ovn-ipsec-azurefile` , is it a bit duplicate, also the job name has some limitation about the length, we'd better make the name as shorter as we can also make it more clearly?

Reply:
Sure, thanks for the double check, let me update it to `azurefile-csi`.